msmbuilder/feature_selection/base.py

```diff
@@ -0,0 +1,119 @@
+# Author: Carlos Xavier Hernandez <cxh@stanford.edu>
```

Comment:
is this code significantly different than the cluster version? Why not just inherent from that directly instead of copying it over.